src/peer-routing/index.js

```diff
@@ -169,7 +173,7 @@ module.exports = (dht) => {
       result.paths.forEach((result) => {
         if (result.success) {
           success = true
-          dht.peerStore.put(result.peer)
+          dht.peerStore.addressBook.add(result.peer.id, result.peer.multiaddrs.toArray())
```

Comment:
I like that this is now a discrete action. It's much clearer what's happening than a `put` imo.